bin/getInfoRippled.sh

```diff
@@ -0,0 +1,47 @@
+ #!/bin/bash
+
+export TMP_LOC="/tmp/ripple_info"
+if [ ! -d "${TMP_LOC}" ]; then
+       sudo mkdir "${TMP_LOC}"
+fi
+
+rm -f "${TMP_LOC}/*"
+cd ${TMP_LOC}
+echo "$ls"
+cd ~
+
+# Send output from this script to a log file
+## this captures any messages
+## or errors from the script itself
+
+export LOG_FILE="${TMP_LOC}/get_info.log"
+exec 3>&1 1>>"${LOG_FILE}" 2>&1
```

Comment:
I've never seen this before, and I think it is very, very cool. Love stuff like this. Having said that, it does complicate the script and I for one am going to do a double take when I see that (most of us on this team are competent scripters, but this seems "next level").

We should consider alternatives before doing this. Maybe a wrapper script for this one that pipes? Maybe don't save the output of this script? I don't know, but I did want to raise the issue.